.github/workflows/run_ert2_test_data_setups.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.6, 3.8]
+        python-version: ['3.6', '3.8', '3.10']
```

Comment:
skipping over `3.7` and `3.9`? I guess there is a reason :)

Reply:
This has been this way for a long time, it just to keep time it takes for testing down, while providing coverage over the python versions :slightly_smiling_face: